utils/checks/snapshots/features.yaml

```diff
@@ -1990,6 +1990,10 @@
   features:
     - default
     - std
+- name: pallet-bridges-initializer-currency-swap 0.1.0
```

Comment:
And this

Reply:
https://github.com/humanode-network/humanode/pull/716/commits/682cb95d89c03e73be78520429e1b72e7eef9f5e